lib/css/exports/_stacks-constants-colors.less

```diff
@@ -276,27 +282,150 @@ body {
     --focus-ring-warning: @focus-ring-warning;
     --focus-ring-error: @focus-ring-error;
     --focus-ring-muted: @focus-ring-muted;
+
+    // Shadows
+    --bs-sm: @bs-sm;
+    --bs-md: @bs-md;
+    --bs-lg: @bs-lg;
+
+    // Scrollbars
+    --scrollbar: @scrollbar;
 }
 
-//  ============================================================================
-//  $   CONSTANT HELPERS
-//      Quick, easy helpers for commonly used colors
-//  --  Background Color Constants
-@bgc-lightest:  @black-050;
-@bgc-lighter:   @black-075;
-@bgc-light:     @black-100;
-@bgc-dark:      @black-200;
-@bgc-darker:    @black-300;
-@bgc-darkest:   @black-350;
-
-//  --  Border Color Constants
-@brc-lightest:  @black-050;
-@brc-lighter:   @black-075;
-@brc-light:     @black-100;
-@brc-dark:      @black-200;
-@brc-darker:    @black-300;
-@brc-darkest:   @black-350;
-@brc-orange:    @orange-400;
+//  --  Dark mode
+.dark-colors() {
+    --white: #2d2d2d;
```

Comment:
Looks like we're not really caring to expose these colors as variables (eg `@dark-black-025`)? I think overall this is the right choice, since we want our users to ultimately consume these colors as `var(--black-025)` anyways and exposing as less variables would be time consuming and make migration harder in the future.